README.md

```diff
@@ -14,15 +14,10 @@ For general questions about using the virtual environments or writing your Actio
 | Ubuntu 18.04 | `ubuntu-18.04` | [ubuntu-18.04] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=ubuntu18&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=ubuntu18&redirect=1)
 | macOS 11 | `macos-11` | [macOS-11] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=macos-11&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=macos-11&redirect=1)
 | macOS 10.15 | `macos-latest` or `macos-10.15` | [macOS-10.15] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=macos-10.15&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=macos-10.15&redirect=1)
+| Windows Server 2022<sup>[beta]</sup> | `windows-2022` | [windows-2022] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2022&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2022&redirect=1) |
 | Windows Server 2019 | `windows-latest` or `windows-2019` | [windows-2019] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2019&redirect=1)
 | Windows Server 2016 | `windows-2016` | [windows-2016] | [![](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&badge=1)](https://actionvirtualenvironmentsstatus.azurewebsites.net/api/status?imageName=windows-2016&redirect=1)
-```
-Ubuntu 16.04 has been deprecated and will be removed on September 20, 2021. Its usage is limited to current customers, no new onboardings are accepted.
-Existing workflows using `Ubuntu 16.04` should migrate to `Ubuntu 20.04` or `Ubuntu 18.04`
-```
-
-> The macOS 11 virtual environment is currently available only on GitHub Actions. Azure DevOps users can track the macOS 11 onboarding progress in [this issue](https://github.com/actions/virtual-environments/issues/2072).<br/>
-The `macos-latest` YAML workflow label still uses the macOS 10.15 virtual environment.
+<b>Note:</b> Beta and Preview images are provided "as-is," "with all faults," and "as available," and are excluded from the service level agreement and warranty. Beta and Preview images may not be covered by customer support.

```

Comment:
Should there be an extra empty line above so this is not part of the table?

![image](https://user-images.githubusercontent.com/4009570/130466889-cd965515-92c7-4e62-ab86-f58e87a41b68.png)

Reply:
Thank you! Good point